Tests/NIOHTTP2Tests/StreamMapTests.swift

```diff
@@ -0,0 +1,191 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+@testable import NIOHTTP2
+
+final class StreamMapTests: XCTestCase {
+    func testContainsSmall() throws {
+        var map = StreamMap<StreamData>()
+
+        // Stuff some stream IDs in. We need even _and_ odd stream IDs, and gaps.
+        for number in stride(from: 1, to: 1000, by: 3) {
+            map.insert(StreamData(number))
+        }
+
+        for number in 1..<1000 {
+            let contains = map.contains(streamID: HTTP2StreamID(number))
+            if number % 3 == 1 {
+                XCTAssertTrue(contains)
```

Comment:
It doesn't really matter but I'd have been tempted to use AssetEquals with the two bools to save a few lines.